client/src/components/admin/student/AddStudent.jsx

```diff
@@ -1,66 +1,186 @@
 import React, { Component } from 'react';
-import { Button, Form, Header } from 'semantic-ui-react';
+import { Button, Form, Header, Input, Message, Icon } from 'semantic-ui-react';
 import PropTypes from 'prop-types';
+import axios from 'axios';
 
-const enrollmentStatus = [
-  { key: 1, text: 'Alum', value: 'alum' },
+const { GHOSTBUSTER_BASE_URL } = process.env;
+
+const enrollmentStatusList = [
+  { key: 0, text: 'Alum', value: 'alum' },
+  { key: 1, text: 'Completed', value: 'completed' },
   { key: 2, text: 'Deferred', value: 'deferred' },
-  { key: 3, text: 'Withdrew', value: 'withdrew' },
-  { key: 4, text: 'Completed', value: 'completed' },
-  { key: 5, text: 'Enrolled', value: 'enrolled' },
-  { key: 6, text: 'Mulligan', value: 'mulligan' },
-  { key: 7, text: 'Removed', value: 'removed' },
-  { key: 8, text: 'Precourse', value: 'precourse' }
+  { key: 3, text: 'Enrolled', value: 'enrolled' },
+  { key: 4, text: 'Mulligan', value: 'mulligan' },
+  { key: 5, text: 'Precourse', value: 'precourse' },
+  { key: 6, text: 'Removed', value: 'removed' },
+  { key: 7, text: 'Withdrew', value: 'withdrew' }
 ];
 
 // eslint-disable-next-line react/prefer-stateless-function
 export default class AddStudent extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      enrollmentId: '',
+      firstName: '',
+      lastName: '',
+      githubHandle: '',
+      zoomName: '',
+      cohort: '',
+      enrollmentStatus: enrollmentStatusList[3].value,
```

Comment:
It's better just to use the actual value here rather than point to a position in an array.  It helps readers know what's going on at a glance and it isn't sensitive to breaking changes if the array is changed to initialize differently.

`enrollmentStatus: 'enrolled';`

Reply:
fixed